app/actions/index.ts

```diff
@@ -11,6 +11,7 @@ export const PAGINATION_CHANGE = "PAGINATION_CHANGE";
 export const UPDATE_MESSAGE = "UPDATE_MESSAGE";
 export const UPDATE_INTERCEPTOR_STATUS = "UPDATE_INTERCEPTOR_STATUS";
 export const FETCH_DATA = "FETCH_DATA";
+export const FETCHING_DATA = "FETCHING_DATA";
```

Comment:
It will be better if we name it `FETCH_DATA_IN_PROGRESS`. 
easy to find the related actions